### ISSUE TITLE
Require local image throws error for Android largeIcon

### ISSUE THREAD

**Issue by Eralyne** [notifee/react-native-notifee#46]
Hey there! 

The library is looking fantastic at the moment but there is one issue that I've run into recently. When using a local image for largeIcon, an error is thrown asking for a string. I've attached incredibly basic code below. This is through the function notifee.displayNotification

```Javascript
android: {
            channelId: 'message',
            timestamp: Date.now(),
            showTimestamp: true,
            largeIcon: require('./Images/logo.png') // This throws the error "notifee.displayNotification(*) 'notification.android.largeIcon' expected a string value."
          },
```

**Comment by Salakar** [comment]
@Eralyne could you try `largeIcon: require('./Images/logo.png').uri` - I think our docs might need updating

**Comment by Eralyne** [comment]
There doesn't seem to be an error thrown anymore but I have yet to test if the icon appears.

**Comment by whenmoon** [comment]
I am still getting the error: 

> Error: notifee.displayNotification(*) 'notification.android.largeIcon' expected a string value

with the code:

```javascript
largeIcon: require('../../assets/large-notification-icon.jpg').uri,
```

**Comment by roni-castro** [comment]
It also happens if you try using a url. I think the types needs to be updated or fixed, as I think this type is not valid for most the Android properties:
<img width="482" alt="Captura de Tela 2020-06-03 às 16 58 19" src="https://user-images.githubusercontent.com/24610813/83683318-92e26780-a5bb-11ea-87ee-2c66f71f60ed.png">


**This works:** 
```
 android: {
      largeIcon: 'https://repository-images.githubusercontent.com/226108661/7d2c3380-1aab-11ea-885e-f3466be45c26'
}
```

**This does not work:**
```
 android: {
      largeIcon: ''
}
```

**This does not work:**
```
 android: {
      largeIcon: undefined
}
```

**Comment by Ehesp** [comment]
Hm yeah, the `| undefined` part is really saying "don't add this key/value pair".

We're checking for [property exists](https://github.com/notifee/react-native-notifee/blob/master/src/validators/validateAndroidNotification.ts#L282), which in this case, it does so it goes into that trap. Maybe we should just convert them to `!isUndefined()`

**Comment by Salakar** [comment]
This has been fixed in 0.6.2 - changelog still to do but theres only 1 breaking change, renaming usages of `Importance` to `AndroidImportance`. Please give it a try, you only need the require now, without `.uri`

Thanks
